CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This gem is moving onto its own [Semantic Versioning](https://semver.org/) schem
 
 Prior to v1.0.0 this gem was versioned based on the `MAJOR`.`MINOR` version of RuboCop. The first release of the ezcater_rubocop gem was `v0.49.0`.
 
+## v2.3.0
+- Add `FeatureFlagActive` cop.
```

Comment:
🎱  Thoughts on noting the reason behind this change to provide extra context to people who are upgrading? This could be especially helpful as someone knows that upgrading rubocop can provide extra confidence when subsequently upgrading ezcater ff.

Reply:
Done